sssd_test_framework/misc/__init__.py

```diff
@@ -17,16 +17,24 @@ def attrs_parse(lines: list[str], attrs: list[str] | None = None) -> dict[str, l
     :rtype: dict[str, list[str]]
     """
     out: dict[str, list[str]] = {}
-    for line in lines:
-        line = line.strip()
+    i = 0
+    while i < len(lines):
+        line = lines[i].strip()
         if not line:
+            i += 1
             continue
 
         (key, value) = map(lambda x: x.strip(), line.split(":", 1))
```

Comment:
use x.lstrip() so we do not loose trailing space. Leading space here is not allowed so that can be stripped.

JFI String that starts with space must be encoded (base64?) and there will be the double colon like this:

    dn:: SFSDIHDKFIEH=

Reply:
ack